dev/Notifications/BadgeNotifications/BadgeNotificationManager.h

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation and Contributors. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include "Microsoft.Windows.BadgeNotifications.BadgeNotificationManager.g.h"
+#include "externs.h"
+#include "BaseNotificationManager.h"
+
+namespace winrt::Microsoft::Windows::BadgeNotifications::implementation
+{
+    struct BadgeNotificationManager : BadgeNotificationManagerT<BadgeNotificationManager>, public ::Microsoft::Windows::BaseNotifications::BaseNotificationManager
+    {
+        BadgeNotificationManager();
+
+        static winrt::Microsoft::Windows::BadgeNotifications::BadgeNotificationManager Current();
```

Comment:
whats the purpose of this method?

Reply:
To get the current instance of BadgeNotificationManager. Only 1 instance of manager will be created

Reply:
We have followed the same patter as AppNotificationManager and PushNotificationManager. Reason is - 
The reason for the AppNotificationManager having a Default API is that it registers with the PushNotificationsLongRunningTask (LRP) which is a singleton process routing ToastNotifications to WinAppSDK consumers. Having multiple AppNotificationManagers would cause multiple registrations with the LRP and would only route ToastNotifications to the most recently instantiated manager.
AppNotificationManager Foo registers with LRP
LRP holds mapping of processName->Foo
AppNotificationManager Bar registers with LRP
LRP overwrites Foo with processName->Bar
Foo is a stale manager doing nothing   
           Our implementation depends on the LRP singleton for managing routing notifications to registered handlers       (App/PushNotificationManager)